how-to-docs/drone-how-to.md

```diff
@@ -356,29 +356,39 @@ The build will fail with the following error:
 [ERROR] 2016/10/20 10:17:40 main.go:130: exit status 1
 ```
 
-This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to DSP.
+This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to ACP.
+
+The kube `deploy.sh` script relies on 3 environment variables:
+
+- `KUBE_NAMESPACE` - The kubernetes namespace you wish to deploy to. **You need to provide the kubernetes namespace as part of the deployment job**.
+
+- `KUBE_TOKEN` - This is the token used to authenticate against the kubernetes cluster. **If you do not already have a kube token, you can [submit a support request for a Kubernetes token on the platform hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/new/kubernetes-token)**.
```

Comment:
Instead of linking to the support request form here, could you point to the "how to" doc instead? This is so if we make it more self-serve in the future we can just put the details in the doc (which is the first point of call).